From: Michal Kubecek <mkubecek@suse.cz>
Date: Tue, 10 Jan 2017 14:30:20 +0100
Subject: kabi: hide changes in struct sk_buff
Patch-mainline: Never, kabi workaround
References: bsc#1009682

Backports of commit 1761f32078ca ("net: Preserve CHECKSUM_COMPLETE at
validation") add members into a hole in a bitfield in struct sk_buff.
Hide these from genksyms.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 include/linux/skbuff.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index a35d7608c510..74801f8a0071 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -550,8 +550,10 @@ struct sk_buff {
 	 * headers if needed
 	 */
 	__u8			encapsulation:1;
+#ifndef __GENKSYMS__
 	__u8			encap_hdr_csum:1; /* unused, for consistency */
 	__u8			csum_valid:1;
+#endif
 	/* 4/6 bit hole (depending on ndisc_nodetype presence) */
 	kmemcheck_bitfield_end(flags2);
 
-- 
2.11.0

